Feature/ds 252 - Updated documentation voice and tone, as well as added some uniformity. (#383)
gsoft-inc/wl-hopper: apps/docs/content/getting-started/javascript.mdx

status: ready
---

If you can't use the default React install, you can still benefit from Hopper UI.

## Tokens

> Hopper [tokens](/tokens) are a collection of design decisions translated into data, you should used them to style your applications

Reviewer: ```suggestion
Hopper [tokens](/tokens) are a collection of design decisions translated into data, which should be used to style your applications.
```

### Installation

Run the following command to install Hopper tokens.

<PackageInstallation library="tokens" mod="prod" />

### Usage

#### CSS

Import the tokens in your project:

```css
@import "@hopper-ui/tokens/tokens.css";
```

> Import dark mode tokens, if your application supports it:

Reviewer: ```suggestion
Import dark mode tokens if your application supports it:
```
The comma would only be needed if the if statement came first.
